src/gui/abstractfileitemwidget.cpp

```diff
@@ -38,10 +38,13 @@ static const int boxVSpacing = 10;
 static const int shadowBlurRadius = 20;
 static const QSize iconSize = QSize(15, 15);
 
+static const QColor folderIconColorLight = QColor("#666666");
+static const QColor folderIconColorDark = QColor("#C0C0C0");
+
 AbstractFileItemWidget::AbstractFileItemWidget(QWidget *parent /*= nullptr*/) : QWidget(parent) {
     setContentsMargins(hMargin, vMargin, hMargin, vMargin);
 
-    QVBoxLayout *mainLayout = new QVBoxLayout(this);
+    auto mainLayout = new QVBoxLayout(this);
```

Comment:
```suggestion
    auto mainLayout = new QVBoxLayout();
```

Reply:
same

Reply:
No, it's not. See the `setLayout` documentation: https://doc.qt.io/qt-6/qwidget.html#setLayout.

Reply:
Done